### PR TITLE
Add a formula for mkdocs

### PR DESCRIPTION
**Pull request by hartzell**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

_You can erase any parts of this template not applicable to your Pull Request._

### New Formulae Submissions:

- [x] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

Add a formula for [mkdocs](https://github.com/mkdocs/mkdocs).

### CODE REVIEW

**Comment by hartzell** [comment]
It seems that it [failed to pass tests on Mavericks](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/version=mavericks/42496/console) because it was missing the `six` prerequisite.

I don't have any mavericks machines to experiment on.  Is this normal?

**Comment by DomT4** [comment]
> I don't have any mavericks machines to experiment on. Is this normal?

Off the top of my head, no idea, but there's no harm adding a `six` resource.

You can use something like [homebrew-pypi-poet](https://github.com/tdsmith/homebrew-pypi-poet) to generate resource blocks for Python formulae semi-automatically. It's available on PyPi but their website seems to be having trouble at the moment.

File: Library/Formula/mkdocs.rb [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    system "#{bin}/mkdocs", "--version"
```
Comment (DomT4) [inline Library/Formula/mkdocs.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by hartzell** [comment]
Thanks for the pointer to poet.  I discovered it after I put this together; I did use it for the [snakemake](https://github.com/Homebrew/homebrew/pull/49606) pull request.

I'll add six and see what happens.

**Comment by hartzell** [comment]
See also #49641

**Comment by hartzell** [comment]
What is the next step for getting this merged?  Thanks!

**Comment by DomT4** [comment]
Mostly just waiting for me to swing back around and merge things. We're maintainer-light at the moment so you're stuck with me, heh.

Merged in https://github.com/Homebrew/homebrew/commit/41efa78e6c674ec450184c05d4aa55b2cbd22b70. Thank you for another contribution to Homebrew @hartzell! :smiley_cat:

**Pull request closed without merging**